SymmetricDifference.java

```diff
@@ -1 +1,31 @@
+package com.company;
 
+import java.util.HashSet;
+import java.util.Set;
+
+public class SymmetricDifference<T> {
+
+    public static <T> Set<T> symmetricDifference(Set<? extends T> set1, Set<? extends T> set2) {
+
+        Set<T> resultSet = new HashSet<>();
+        resultSet.addAll(set1);
+        resultSet.addAll(set2);
+        Iterator itr = resultSet.iterator();
+        while (itr.hasNext()) {
```

Comment:
optional: we can write it without while cycle, with using method retainAll() and removeAll()